yt_dlp/extractor/facebook.py

```diff
@@ -500,6 +531,7 @@ def extract_metadata(webpage):
                 webpage, 'description', default=None)
             uploader_data = (
                 get_first(media, ('owner', {dict}))
+                or get_first(post, ('video', 'creation_story', 'attachments', ..., 'media', lambda k, v: k == 'owner' and v['name']))
```

Comment:
Does this fix #9057?

Reply:
I'm not sure. It's more of a case where `uploader` was not filled at all I believe. But I don't remember exactly at this point to be frank.

Reply:
Can confirm it doesn't fix that.